src/components/layout/PanelChange/index.tsx

```diff
@@ -234,7 +235,7 @@ export const PanelChange: FC<IProps> = ({
 										<span className={styles.number}>
 											{`$${personalFlow && getFlowUSDValue(personalFlow)} ${t('per month')}`}
 										</span>
-										{(subsidyRate?.perso || 0) > 0 ? (
```

Comment:
I think you adjusted the wrong subsidy, `subsidyRate?.perso` is the amount the user is earning, its a calculation that takes the subsidy rate and divides it by their fraction of the pool
<img width="1682" alt="Screen Shot 2022-07-24 at 11 03 22 AM" src="https://user-images.githubusercontent.com/5077880/180653251-8a6caf83-b43e-473f-a143-e4407e6d8f05.png">

Reply:
Look at line 359

Reply:
It would be great if you could get subsidy rate perso working, so it shows their personal RIC per month, hard code something somewhere as 50K per month